packages/cache/src/internal/constants.ts

```diff
@@ -5,6 +5,7 @@ export enum CacheFilename {
 
 export enum CompressionMethod {
   Gzip = 'gzip',
+  ZstdOld = 'zstd-old',
```

Comment:
Could we add a comment or be more descriptive in the enum name for what's considered an "Old" version of zstd?

The latest version of zstd is [v1.4.4](https://github.com/facebook/zstd/releases/tag/v1.4.4), so both of the versions we're dealing with could be considered old.